tests/Makefile

```diff
@@ -19,79 +16,108 @@ all: minikube operator test
+	@helm upgrade --install k8s-agents-operator ${LOCAL_OPERATOR_REPO_PATH}/charts/k8s-agents-operator/ \
 		--set=licenseKey=${NEW_RELIC_LICENSE_KEY} \
 		--set=newRelicHost=${NEW_RELIC_HOST} \
 		--set=controllerManager.manager.image.pullPolicy=Never \
 		--set=controllerManager.manager.image.repository=e2e/k8s-agents-operator \
 		--set=controllerManager.manager.image.version=e2e
-	sleep 5
-	kubectl wait --for=condition=Ready pods $$(kubectl get pods | grep k8s-agents-operator | cut -d" " -f1)
+	@sleep 5
+	@kubectl wait --for=condition=Ready pods $$(kubectl get pods | grep k8s-agents-operator | cut -d" " -f1)
+	@echo "===== Giving operator time to generate certificates. ====="
+	@sleep 30
 
 # Set license key in default namespace
 .PHONY: license_key_secret
 license_key_secret:
-	kubectl delete secret newrelic-key-secret --ignore-not-found
-	kubectl create secret generic newrelic-key-secret \
+	@echo "===== Setting license key secret. ====="
+	@kubectl delete secret newrelic-key-secret --ignore-not-found
+	@kubectl create secret generic newrelic-key-secret \
 		--from-literal="new_relic_license_key=${NEW_RELIC_LICENSE_KEY}"
 
 # Build local initcontainer image
 .PHONY: build-initcontainer
 build-initcontainer: check-language-arg
-	eval $$(minikube docker-env --shell=bash) && \
+	@echo "===== Building initcontainer image. ====="
+	@eval $$(minikube docker-env --shell=bash) && \
 		docker build -t e2e/newrelic-${INITCONTAINER_LANGUAGE}-init:e2e ${REPO_ROOT}/src/${INITCONTAINER_LANGUAGE}/
 
 # Build local test app image
 .PHONY: build-testapp
 build-testapp: check-language-arg
-	eval $$(minikube docker-env --shell=bash) && \
+	@echo "===== Building test app container image. ====="
+	@eval $$(minikube docker-env --shell=bash) && \
 		docker build -t e2e/test-app-${INITCONTAINER_LANGUAGE}:e2e ${REPO_ROOT}/tests/${INITCONTAINER_LANGUAGE}/
 
 # Deploy and open test app in browser
 .PHONY: test
 test: build-initcontainer build-testapp check-language-arg
-	helm uninstall test-deployment --ignore-not-found
-	sleep 1
-	helm install test-deployment ${REPO_ROOT}/tests/${INITCONTAINER_LANGUAGE}/chart/
-	sleep 3
-	kubectl wait --for=condition=Ready -n default --all pods
-	minikube service test-app-${INITCONTAINER_LANGUAGE}-service -n default
+	@echo "===== Removing any existing test app deployments. ====="
+	@helm uninstall test-deployment --ignore-not-found
+	@sleep 1
+	@echo "===== Deploying test app to minikube cluster. ====="
+	@helm install test-deployment ${REPO_ROOT}/tests/${INITCONTAINER_LANGUAGE}/chart/
+	@sleep 5
+	@kubectl wait --for=condition=Ready -n default --all pods
+	@echo "===== Recreating test app pod to ensure instrumentation resource is used. ====="
+	@kubectl delete pods -l "app=test-app-${INITCONTAINER_LANGUAGE}"
+	@sleep 5
+	@kubectl wait --for=condition=Ready -n default --all pods
+# If yq is installed, check for the initcontainer to be visible on the pod spec
+	@which -s yq && \
```

Comment:
@TimPansino - I got stuck on this step because `yq` wasn't installed on my machine. 

I installed it via brew using `brew install yq`.

Is this something that should be added to the README? Should it have been installed with any of the previous steps in the Makefile?

Reply:
Ah that's a tiny bug. I was trying to make it so that that step would only happen if yq was installed, but I messed up the scripting slightly.

Reply:
I'll fix that up for anyone without yq in a fast follow up PR so that we can get the CI unblocked first.